### PR TITLE
#157110617 Add Fitbit Support

### PR DESCRIPTION
**Pull request by ghost**
#### What does this PR do?
Add Fitbit support to enable a user to sync their data from Fitbit into wger.
#### Description of Task to be completed?
As a user, I should be able to pull in weight information from Fitbit after authorizing wger to access my Fitbit account. 
#### How should this be manually tested?
After logging into wger click on the `Sync Fitbit` button then login to you Fitbit account.
#### What are the relevant pivotal tracker stories?
[#157110617](https://www.pivotaltracker.com/story/show/157110617)

### CODE REVIEW

File: wger/core/views/fitbit.py [hunk 1]
```diff
@@ -1,31 +1,38 @@
...
-    TOKEN_REQUEST_URI = os.getenv('TOKEN_REQUEST_URI')
+    """Class to handle all fitbit operation"""
+    # App settings from fitbit as regards the app
+    CLIENT_ID = os.getenv('CLIENT_ID', 'DEFAULT_VALUE')
+    CLIENT_SECRET = os.getenv('CLIENT_SECRET', 'DEFAULT_VALUE')
+    SCOPE = os.getenv('SCOPE', 'DEFAULT_VALUE')
+    REDIRECT_URI = os.getenv('REDIRECT_URI', 'DEFAULT_VALUE')
+    # Authorization and authentication URIs
+    AUTHORIZE_URI = os.getenv('AUTHORIZE_URI', 'DEFAULT_VALUE')
+    TOKEN_REQUEST_URI = os.getenv("TOKEN_REQUEST_URI", 'DEFAULT_VALUE')
```
Comment (MarloneA) [inline wger/core/views/fitbit.py]: Great work setting up Fitbit

Comment (mirr254) [reply wger/core/views/fitbit.py]: I like the way you have commented your code. Makes it easy to understand.

File: Procfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+web: gunicorn wger.wsgi --log-file -
+invoke create-settings --settings-path /wger/settings.py --database-path /wger/database.sqlite
```
Comment (Rodgers-M) [inline Procfile]: good work having heroku invoke database creation.

**Review by younggeeks — COMMENTED** [review]
LGTM

**Pull request closed without merging**